tests/integration/ui_write/base.py

```diff
@@ -194,6 +202,10 @@ def getContainer(self, nwbfile):
         ''' Should take an NWBFile object and return the Container'''
         raise unittest.SkipTest('Cannot run test unless getContainer is implemented')
 
+    def actOnContainer(self, nwbfile):
```

Comment:
Is the point of this to provide the ability to do something to a container after roundtripping?